make/autoconf/buildjdk-spec.gmk.in

```diff
@@ -103,3 +103,7 @@ JVM_FEATURES_server := cds compiler1 compiler2 g1gc serialgc
 override EXTRA_CFLAGS :=
 override EXTRA_CXXFLAGS :=
 override EXTRA_LDFLAGS :=
+
+# hsdis is not needed
+override HSDIS_BACKEND := none
+override ENABLE_HSDIS_BUNDLING := false
```

Comment:
You shouldn't need the `override` keyword. That is only needed when the value of the variables is set on the make command line. In this case we are simply overriding the values from the main `spec.gmk`.

Reply:
updated.